### ISSUE TITLE
Calendar Bug with Navigator selected wrong Month

### ISSUE THREAD

**Issue by cyberrranger** [primefaces/primeng#6114]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
```

**Case (Bug Reports)**

https://www.primefaces.org/primeng/#/calendar use Navigator

=> the wrong month is selected

* **Angular version:** 6.0.1
* **PrimeNG version:** 6.x

**Comment by cagataycivici** [comment]
#6118

**Comment by cyberrranger** [comment]
@cagataycivici sry, but is not fixed in 6.0.2 and nor in 6.1.0-RC1

<p-calendar [(ngModel)]="date6" [monthNavigator]="true" [yearNavigator]="true" yearRange="2000:2030"></p-calendar>

if date6 == null, => the month is the current month and the year is the current year, 
but if date6 != null, example 2019-09-06 the month and year are wrong.

![screenshot 2018-08-06 11 55 43](https://user-images.githubusercontent.com/21671762/43710375-c039b4ae-996f-11e8-9c9a-36e77fa21019.png)
